ui/components/Button.js

```diff
@@ -1,8 +1,30 @@
-import Button from '@mui/material/Button';
+import{ Button, Box} from '@mui/material/';
 import { styled } from "@mui/material/styles";
 import { deepSpaceSparkle } from '../styles/colors';
 
 export const MetricsButton = styled(Button)(({theme})=>({
     backgroundColor: deepSpaceSparkle,
-    width: "240px",
 }))
+
+
+ const RightAlignButtonContainer = styled(Box)(({theme})=>({
```

Comment:
`Container` is being used differently in our codebase, let's use a different name for this purpose. Maybe `wrapper`?